### PR TITLE
Add detailed support center info

### PR DESCRIPTION
**Pull request by matyasselmeci**
The webapp also needed modification to handle the new format. Tested on topology-itb.

### CODE REVIEW

**Comment by matyasselmeci** [comment]
After merging, **immediately update** `/opt/topology` on `topology.opensciencegrid.org` and restart httpd, or things will break.

File: src/app.py [hunk 1]
```diff
@@ -150,6 +150,11 @@ def get_projects(self) -> Dict:
 if "TOPOLOGY_CONFIG" in os.environ:
     app.config.from_envvar("TOPOLOGY_CONFIG", silent=False)
 _verify_config(app.config)
+if "AUTH" in app.config:
+    if app.debug:
+        default_authorized = app.config["AUTH"]
+    else:
+        print("ignoring AUTH option in non-debugging mode", file=sys.stderr)
```
Comment (brianhlin) [inline src/app.py]: Should we use `app.logger` here instead of `print`?

Comment (matyasselmeci) [reply src/app.py]: Logging hasn't been initialized at that point in the code, so that wouldn't work.

Comment (brianhlin) [reply src/app.py]: So does this get logged in the journal (assuming it's run as a systemd service)?

Comment (matyasselmeci) [reply src/app.py]: Yeah, it ends up in the httpd logs.

**Comment by brianhlin** [comment]
@matyasselmeci let's get this merged and done today

**Pull request merged**